.github/workflows/maven.yml

```diff
@@ -0,0 +1,42 @@
+name: Maven Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        os: [ubuntu-24.04, macOS-13, windows-2022]
```

Comment:
The `macOS-13` runner has been removed. Please replace `macOS-13` with both `macOS-15-intel` and `macOS-15` (Apple Silicon/aarch64).